app/src/main/java/ju/video/player/view/commons/ComboBox.java

```diff
@@ -21,19 +25,29 @@
  */
 
 public class ComboBox extends JComboBox<String> implements ItemListener {
-    
+
     public ComboBox(String[] itemsList) {
+        addItem("All formats");
         addItems(itemsList);
         addItemListener(this);
+        addItemListener(new ItemListener() {
+            public void itemStateChanged(ItemEvent event) {
+                String itemSelected = (String) event.getItem();
+                if (itemSelected.equals("All formats")) {
+                    itemSelected = null;
+                }
+                ListValidVideos.getInstance().setFormatSelected(itemSelected);
+            }
+        });
     }
 
     /**
      * Add a list of items to the comboBox
-     * 
+     *
      * @param itemsList
```

Comment:
You could put the description of the ItemsList parameter

Reply:
done